lib/k8s/client/runner/base.ex

```diff
@@ -86,10 +86,36 @@ defmodule K8s.Client.Runner.Base do
     do: run(conn, operation, [])
 
   @doc """
-  Run an operation and pass `http_opts` to `K8s.Client.HTTPProvider`
-  See `run/2`
+  Run a connect operation and pass `websocket_driver_opts` to `K8s.Client.WebSocketProvider`
+  See `run/3`
   """
   @spec run(Conn.t(), Operation.t(), keyword()) :: result_t
+  def run(%Conn{} = conn, %Operation{verb: :connect} = operation, websocket_driver_opts) do
+    Process.flag(:trap_exit, true)
```

Comment:
It's probably better to let the caller decide if they want to trap exits or not.